internal_testsuites/golang/testsuite/startosis_start_service_test/startosis_start_service_test.go

```diff
@@ -42,17 +37,19 @@ def run(plan):
 	plan.add_service(name = DATASTORE_SERVICE_NAME, config = config)
 	plan.print("Service " + DATASTORE_SERVICE_NAME + " deployed successfully.")
 `
-	// We start the service we created through the script above with a different script
-	startScript = `
+	
+	// We stop and restart the service we created through the script above with a different script
+	stopAndStartScript = `
```

Comment:
Test with a stop service followed by a start service instruction in the same Starlark script.